src/modules/addToDo.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
```

Comment:
- [ ] @HunterX-7 kindly remove these two lines, it's not good practice to disable the linter in your entire file, try and resolve the linter errors instead of disabling them